com.unity.netcode.gameobjects/Tests/Runtime/Metrics/RttMetricsTests.cs

```diff
@@ -0,0 +1,103 @@
+#if MULTIPLAYER_TOOLS
+#if MULTIPLAYER_TOOLS_1_0_0_PRE_3
+
+using System.Collections;
+using System.Linq;
+using NUnit.Framework;
+using Unity.Collections;
+using Unity.Multiplayer.Tools.MetricTypes;
+using Unity.Netcode.RuntimeTests.Metrics.Utility;
+using UnityEngine.TestTools;
+
+namespace Unity.Netcode.RuntimeTests.Metrics
+{
+    internal class RttMetricsTests
+    {
+        [UnityTest]
+        public IEnumerator TrackRttMetricServerToSingleClient()
+        {
+            MultiInstanceHelpers.Create(
+                clientCount: 1,
+                out var server,
+                out var clients,
+                targetFrameRate: 60,
+                MultiInstanceHelpers.InstanceTransport.UTP);
+
+            server.StartServer();
+            clients[0].StartClient();
+
+            var serverMetrics = (NetworkMetrics)server.NetworkMetrics;
+            var waitForMetricValues = new WaitForGaugeMetricValues(serverMetrics.Dispatcher, NetworkMetricTypes.Rtt);
+
+            using (var writer = new FastBufferWriter(sizeof(uint), Allocator.Temp))
+            {
+                writer.WriteValueSafe(1337);
+                server.CustomMessagingManager.SendUnnamedMessageToAll(writer);
+            }
+
+            yield return waitForMetricValues.WaitForMetricsReceived();
+
+            var rttValue = waitForMetricValues.AssertMetricValueHaveBeenFound();
+            Assert.That(rttValue, Is.EqualTo(0f));
```

Comment:
Assert.AreEqual(0f, rttValue); is more common and descriptive here

Reply:
Also more concise